### PR TITLE
Added support for quoted identifiers for install-check

### PR DESCRIPTION
**Pull request by ferrarius27**
For database names containing  arbitrary sequence of characters e.g. TEST"ME,TEST ME,etc.

### CODE REVIEW

File: src/madpack/madpack.py [hunk 1]
```diff
@@ -674,7 +674,6 @@ def _process_py_sql_files_in_modules(modset, args_dict):
 
         # Loop through all SQL files for this module
         source_files = glob.glob(mask)
-        source_files = [s for s in source_files if '.setup' not in s]
```
Comment (orhankislal) [inline src/madpack/madpack.py]: This line was included to ensure the common setup files for tests (like `madlib_keras_cifar.setup.sql_in`) are not treated as individual tests.

Comment (ferrarius27) [reply src/madpack/madpack.py]: Sorry for the late response,looks like this line was accidentally deleted,it has nothing to do with the fix

Comment (orhankislal) [reply src/madpack/madpack.py]: No worries, I removed this line before merging the code. Thanks for the PR.

Comment (ferrarius27) [reply src/madpack/madpack.py]: Ok,thanks.

**Pull request closed without merging**